gradle.properties

```diff
@@ -3,13 +3,13 @@
 
 pluginGroup = com.phrase.intellij
 pluginName = Phrase
-pluginVersion = 5.1.0
+pluginVersion = 5.1.1
 pluginSinceBuild = 212
-pluginUntilBuild = 213.*
+pluginUntilBuild = 221.*
```

Comment:
Took a quick look at their docs and it seems 223 is already available. Should we bump this number a bit more?

Reply:
probably it would be safe, yes. I just tried it with latest Android Studio, which is 221. Will check the IDEA as well.